packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -645,7 +645,7 @@ export class FixieCorpus<ChunkMetadata extends Jsonifiable = Jsonifiable> implem
       body: JSON.stringify({
         corpus_id: this.corpusId,
         query,
-        page_size: params?.limit ?? 10,
+        max_chunks: params?.limit ?? 10,
       }),
     });
     if (response.status !== 200) {
```

Comment:
Could you update line 661 to use result.chunkContent while you're here please?

Reply:
done